test/test_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV['RACK_ENV'] = 'test'
+require 'pry'
```

Comment:
Assuming this is here intentionally I think pry_byebug would be better than pry.